### DISCUSSION TITLE
Unscrollable PDF

### DISCUSSION THREAD

**Discussion by USC-MB288** [RyotaUshio/obsidian-pdf-plus#213] [Q&A]
I have toggled the option to make it so pdfs that have a page specified can't be scrolled (amazing because it is so annoying when it scrolls). Is there a way I can specify multiple pages and only that range can be scrolled

**Comment by RyotaUshio** [comment]
Unfortunately, it's not supported for now. But the idea sounds nice so I'll add it if I can figure out how to implement it.
